examples/LA/003_LA_ORU_R01_NBS_3_hl7_translation_final_obr_flatten_ex_1.hl7

```diff
@@ -0,0 +1,56 @@
+MSH|^~\&|Natus^natus.health.state.la.us^DNS|Louisiana Office of Public Health Laboratory^2.16.840.1.114222.4.3.26.3.1^ISO|Test Department of tests^1.2.840.114350.1.13.286.2.7.2.695071^ISO|^simulated-hospital-id^L,M,N|20240101050000+0000||ORU^R01^ORU_R01|20240101050000_0001|T|2.5.1|||AL|AL|||||LRI_NG_FRN_PROFILE^^2.16.840.1.113883.9.195.3.4^ISO~LRI_NDBS_COMPONENT^^2.16.840.1.113883.9.195.3.6^ISO~LAB_PRN_Component^^2.16.840.1.113883.9.81^ISO~LAB_TO_COMPONENT^^2.16.840.1.113883.9.22^ISO
```

Comment:
Mentioned this in a comment for another file, MSH-11 should be updated to `D` here as well